### PR TITLE
 transmission-web-control: add new package

### PR DESCRIPTION
**Pull request by ysc3839**
Maintainer: me
Compile tested: [openwrt-sdk-sunxi-cortexa7_gcc-7.4.0_musl_eabi.Linux-x86_64.tar.xz](https://downloads.openwrt.org/snapshots/targets/sunxi/cortexa7/openwrt-sdk-sunxi-cortexa7_gcc-7.4.0_musl_eabi.Linux-x86_64.tar.xz)
Run tested: no

Description:
Transmission Web Control is a custom web UI for Transmission.
https://github.com/ronggang/transmission-web-control/wiki

### CODE REVIEW

**Comment by BKPepe** [comment]
Not sure, if we need it. There is already support for other custom web ui for Transmission.
https://github.com/openwrt/packages/pull/6462 Some time ago, I tested it with that PR and it was working.

**Comment by ysc3839** [comment]
@BKPepe However there's some aria2 webui packages.
https://github.com/openwrt/packages/tree/master/net/ariang
https://github.com/openwrt/packages/tree/master/net/webui-aria2
https://github.com/openwrt/packages/tree/master/net/yaaw

File: net/transmission-web-control/Makefile [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+define Package/transmission-web-control/description
+ Transmission Web Control is a custom Web UI for Transmission.
+endef
+
+define Build/Compile
+endef
+
+define Package/transmission-web-control/install
+	$(INSTALL_DIR) $(1)/usr/share/transmission/web
+	$(CP) $(PKG_BUILD_DIR)/src/* $(1)/usr/share/transmission/web
```
Comment (neheb) [inline net/transmission-web-control/Makefile]: Could probably be replaced with INSTALL_DATA

**Comment by neheb** [comment]
I'm open to merging this. The stock UI has had several bugs historically. One of those has a patch included in OpenWrt.

File: net/transmission-web-control/Makefile (hunk 1 above)
Comment (ysc3839) [reply net/transmission-web-control/Makefile]: It seems that INSTALL_DATA can't do recursive copy.

**Pull request merged**